scan-irb/example-params.csv

```diff
@@ -1,7 +1,7 @@
-qrcode,pat_name,birth_date,collect_ts,result_ts,status_code
-AAAAAAAA,Not-received Ned,2020-01-01,,,not-received
-BBBBBBBB,Pending Patrick,2020-01-01,2020-06-10,,pending
-CCCCCCCC,Never-tested Nellie,2020-01-01,2020-06-10,,never-tested
-DDDDDDDD,Negative Nancy,2020-01-01,2020-06-10,2020-06-12,negative
-EEEEEEEE,Inconclusive Ian,2020-01-01,2020-06-10,2020-06-12,inconclusive
-FFFFFFFF,Positive Pat,2020-01-01,2020-06-10,2020-06-12,positive
+qrcode,pat_name,birth_date,collect_ts,result_ts,status_code,swab_type,staff_collected
+AAAAAAAA,Not-received Ned,2020-01-01,,,not-received,ans,0
+BBBBBBBB,Pending Patrick,2020-01-01,2020-06-10,,pending,mts,0
```

Comment:
Was the idea that the view that produces the results would do the translating and the template wouldn't have to? That is, would we see "mtb" in the CSV file or "Midnasal"?

Reply:
Good thing to consider. I think it could go either way. If the view does translate, then we need to make sure that unknown/null values from the view ends up causing those reports to fail. By translating in the template, we kinda get that assurance as a side-effect.